src/SocialAuthUserManager.php

```diff
@@ -363,7 +359,7 @@ public function addUserRecord($user_id, $pluginId, $provider_user_id) {
           array(
```

Comment:
array() syntax must be replaced by [] due to a new coding standard. Please change all the matches in this file.

[Leave the code cleaner than you found it](http://pragmaticcraftsman.com/2011/03/the-boy-scout-rule/)